ols/app/models/config.py

```diff
@@ -96,104 +97,61 @@ class InvalidConfigurationError(Exception):
 class ModelConfig(BaseModel):
     """Model configuration."""
 
-    name: Optional[str] = None
+    name: str
 
     # TODO: OLS-656 Switch OLS operator to use provider-specific configuration parameters
     url: Optional[AnyHttpUrl] = None
     credentials: Optional[str] = None
 
-    context_window_size: int = -1  # need to be set later, based on model
-    max_tokens_for_response: int = -1  # need to be set later, based on model
+    context_window_size: Optional[PositiveInt] = None
```

Comment:
it is not optional, it is `constants.DEFAULT_CONTEXT_WINDOW_SIZE` by default

Reply:
Right.